package.json

```
@@ -17,7 +18,7 @@
   },
   "homepage": "https://github.com/nteract/kernelspecs#readme",
   "dependencies": {
-    "jupyter-paths": "^2.0.0"
+    "jupyter-paths": "^2.0.3"
```

```suggestion
    "jupyter-paths": "2.0.3"
```

Since we don't use lock files in this repo let's just pin this to a certain version to make things predictable.

---

Yes, we can pin it for now.

We should update the version once the following PR is registered.
https://github.com/nteract/jupyter-paths/pull/39